src/sentry/event_manager.py

```diff
@@ -146,8 +146,8 @@
 
 HIGH_SEVERITY_THRESHOLD = 0.1
 
-SEER_GLOBAL_RATE_LIMIT_DEFAULT = 20  # 20 requests per second
-SEER_PROJECT_RATE_LIMIT_DEFAULT = 5  # 5 requests per second
+SEER_GLOBAL_RATE_LIMIT_DEFAULT = 1200  # 1200 requests per minute
+SEER_PROJECT_RATE_LIMIT_DEFAULT = 300  # 300 requests per minute
```

Comment:
@snigdhas 300 seems rather high to me

Reply:
What would you suggest? right now, it's the same limit as before but spread out over a minute